build/publish.proj

```diff
@@ -25,43 +30,33 @@
 
     <Error Condition="'@(ItemsToPush)' == ''" Text="No packages to push." />
 
-    <ItemGroup>
-      <BuildData Include="Location=$(FeedUrl)" />
-    </ItemGroup>
-
-    <Error Condition="'$(AccountKey)' == ''" Text="The AccountKey property is required." />
-    <Error Condition="'$(BuildId)' == ''" Text="The BuildId property is required." />
-    <Error Condition="'$(ManifestFilePath)' == ''" Text="The ManifestFilePath property is required." />
-    <Error Condition="'$(RepoBranch)' == ''" Text="The RepoBranch property is required." />
-    <Error Condition="'$(RepoCommit)' == ''" Text="The RepoCommit property is required." />
-    <Error Condition="'$(RepoUri)' == ''" Text="The RepoUri property is required." />
+    <Error Condition="'$(BUILD_BUILDNUMBER)' == ''" Text="The BUILD_BUILDNUMBER property is required." />
+    <Error Condition="'$(ArtifactsLogDir)' == ''" Text="The ArtifactsLogDir property is required." />
+    <Error Condition="'$(BUILD_SOURCEBRANCH)' == ''" Text="The BUILD_SOURCEBRANCH property is required." />
+    <Error Condition="'$(BUILD_SOURCEVERSION)' == ''" Text="The BUILD_SOURCEVERSION property is required." />
+    <Error Condition="'$(BUILD_REPOSITORY_URI)' == ''" Text="The BUILD_REPOSITORY_URI property is required." />
+    <Error Condition="'$(MaestroAccessToken)' == ''" Text="The MaestroAccessToken property is required." />
+    <Error Condition="'$(MaestroApiEndpoint)' == ''" Text="The MaestroApiEndpoint property is required." />
 
     <Message Text="Publishing %(ItemsToPush.Identity)" Importance="normal" />
 
-    <PushToBlobFeed
-      AccountKey="$(AccountKey)"
-      AssetManifestPath="$(ManifestFilePath)"
-      ExpectedFeedUrl="$(FeedUrl)"
+    <PushToAzureDevOpsArtifacts
       ItemsToPush="@(ItemsToPush)"
-      ManifestBranch="$(RepoBranch)"
-      ManifestBuildId="$(BuildId)"
-      ManifestBuildData="$(BuildData)"
-      ManifestCommit="$(RepoCommit)"
-      ManifestRepoUri="$(RepoUri)"
-      MaxClients="8"
-      Overwrite="false"
-      PassIfExistingItemIdentical="true" />
-  </Target>
+      ManifestBuildData="Location=$(FeedUrl)"
+      ManifestRepoUri="$(BUILD_REPOSITORY_URI)"
+      ManifestBranch="$(BUILD_SOURCEBRANCH)"
+      ManifestBuildId="$(BUILD_BUILDNUMBER)"
+      ManifestCommit="$(BUILD_SOURCEVERSION)"
+      AssetManifestPath="$(AssetManifestPath)" />
 
-
-  <Target Name="PublishManifestToBuildAssetRegistry">
-    <Error Condition="'$(MaestroAccessToken)' == ''" Text="The MaestroAccessToken property is required." />
-    <Error Condition="'$(MaestroApiEndpoint)' == ''" Text="The MaestroApiEndpoint property is required." />
-    <Error Condition="!Exists($(ManifestsDirectoryPath))" Text="The manifests directory path '$(ManifestsDirectoryPath)' does not exist." />
+    <Message
```

Comment:
Actually, you don't need this line anymore. I understand the documentation shows it, i'll fix the docs. Sorry about the confusion.

Reply:
The docs also say to use `AssetsTemporaryDirectory`, but when I did I got a message telling me to stop, so that'd be another thing nice to fix. Also the `Location=` change that you mentioned in another comment.

Reply:
Thanks Andy. I'll fix that too.